openrtb-core/src/main/protobuf/openrtb.proto

```diff
@@ -2671,6 +2671,13 @@ enum NoBidReason {
   UNMATCHED_USER = 8;
   DAILY_READER_CAP = 9;
   DAILY_DOMAIN_CAP = 10;
+  TTD_ADS_TXT_AUTH_VIOLATION = 12;
```

Comment:
Thoughts on why we're using this as an enum instead of a simple int? In theory, bidders could have overlapping meanings for certain numbers, meaning an enum would be misleading. Also if this is omitting other valid values, it's only making this harder.

Reply:
Agree. But that's how the original openrtb-core library works.  Do you suggest further customization? As I get it, we're trying to be as close to the original as possible to the extent that at some point we will be able to get rid of the custom one at all.